### PR TITLE
Add reference architecure documentation

### PR DESCRIPTION
**Pull request by mkrudele**
### Description

Added reference architecture documentation (SF UX025).  

### Release required?

- [X] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by andreainnocenti — APPROVED** [review]

**Review by vburckhardt — DISMISSED** [review]

File: reference-architectures/watsonx-saas-da.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+The Watsonx AI SaaS Automation with Assistant and Governance adheres to the IBM Cloud for Financal Services Security & Compliance Center profile.
+
+## Next steps
+{: #next-steps}
+
+_Optional section._ Include links to your deployment guide or next steps to get started with the architecture.
+
+
+:exclamation: **Important:** Rename this file `<architecture-name>.md`. For deployable architectures, `<architecture-name>` is the same as the deployable architecture name.
```
Comment (andreainnocenti) [inline reference-architectures/watsonx-saas-da.md:155]: should you remove this line?

**Review by hlucey — CHANGES_REQUESTED** [review]

File: reference-architectures/watsonx-saas-da.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+<!--
+The IDs, such as {: #title-id} are required for publishing this reference architecture in IBM Cloud Docs. Set unique IDs for each heading. Also include
+the toc attributes on the H1, repeating the values from the YAML header.
+ -->
+
+The Watsonx AI SaaS deployable architecture is designed to automate the deployment and
+configuration of the IBM watsonx platform in an IBM Cloud account.
+The IBM watsonx platform is made of several services working together to offer AI capabilities to end users,
+who can explore them using [IBM watsonx projects](https://dataplatform.cloud.ibm.com/docs/content/wsj/manage-data/manage-projects.html?context=wx&audience=wdp).
+On that purpose, the automation also configures a *starter project* for an existing IBM Cloud user.
```
[the remark this answers is not in the document]
Comment (mkrudele) [reply reference-architectures/watsonx-saas-da.md:87]: Why {{site.data.keyword.IBM}} and {{site.data.keyword.Bluemix_notm}} are different? Shouldn't be the same? And which one to use?

File: reference-architectures/watsonx-saas-da.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+![Design requirements for Watsonx AI SaaS deployable architecture](heat-map.svg "Design requirements"){: caption="Figure 2. Scope of the design requirements" caption-side="bottom"}
+
+## Requirements
+{: #requirements}
+
+The following table outlines the requirements that are addressed in this architecture.
+
+| Aspect | Requirements |
+| -------------- | -------------- |
+| Enterprise applications | Lay down and grant access to the {{site.data.keyword.IBM_notm}} Watsonx Artificial Intelligence and Governance platform. |
```
Comment (hlucey) [inline reference-architectures/watsonx-saas-da.md:118]: What do we mean by lay down here?

File: reference-architectures/watsonx-saas-da.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+{: caption="Table 1. Requirements" caption-side="bottom"}
+
+## Components
+{: #components}
+
+The following table outlines the services used in the architecture for each aspect.
+
+| Aspects | Architecture components | How the component is used |
+| -------------- | -------------- | -------------- |
+| Storage | Cloud Object Storage | Stores artificial intelligence {{site.data.keyword.IBM_notm}} Watsonx data assets managed by Watson Machine Learning and Watson Studio services. |
```
Comment (hlucey) [inline reference-architectures/watsonx-saas-da.md:132]: Should Watsonx be upper or lowercase? Seeing different variations.

File: reference-architectures/watsonx-saas-da.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+for wiring the deployable architecture to the other components of the stack, and it implements the flexibility
+to install additional Watson services.
+
+## Architecture diagram
+{: #architecture-diagram}
+
+![Architecture diagram for the Watsonx AI SaaS Automation with Assistant and Governance deployable architecture](watsonx-saas-da.svg "Architecture diagram for the Watsonx AI SaaS Automation with Assistant and Governance deployable architecture")
+{: caption="Figure 1. Watsonx AI SaaS Automation with Assistant and Governance deployable architecture" caption-side="bottom"}{: external download="watsonx-saas-da.svg"}
+
+The Watsonx AI SaaS deployable architecture lays down the services shown in the WatsonX services box and an instance of {{site.data.keyword.cos_full_notm}} in a target {{site.data.keyword.Bluemix_notm}} account, resource group, and region. Then, it automatically configures a starter project that grants access to an existing {{site.data.keyword.Bluemix_notm}} user, for example, an AI researcher. As a result, that user can
```
[the remark this answers is not in the document]
Comment (mkrudele) [reply reference-architectures/watsonx-saas-da.md:101]: Yes

File: reference-architectures/watsonx-saas-da.md (hunk 4 above)
Comment (mkrudele) [reply reference-architectures/watsonx-saas-da.md:132]: When referred to the watsonx should be always `IBM watsonx`.  
When referred to classic watson services should be Watson. I'll fix.

File: reference-architectures/watsonx-saas-da.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+| Storage | Cloud Object Storage | Stores Artificial intelligence IBM Watsonx data assets managed by Watson Machine Learning and Watson Studio services |
+| Security | IAM | IBM Cloud Identity & Access Management authenticates and authorizes any user interaction |
+| Resiliency | All IBM Cloud provisioned services | Fully managed services that provide resiliency and high availability |
+| Application platforms | IBM watsonx platform | End users interact with the IBM watsonx platform to manage Artificial intelligence assets and data |
+{: caption="Table 2. Components" caption-side="bottom"}
+
+## Compliance
+{: #compliance}
+
+The Watsonx AI SaaS Automation with Assistant and Governance adheres to the IBM Cloud for Financal Services Security & Compliance Center profile.
```
[the remark this answers is not in the document]
Comment (mkrudele) [reply reference-architectures/watsonx-saas-da.md:141]: done

**Review by andreainnocenti — DISMISSED** [review]

File: reference-architectures/watsonx-saas-da.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+A typical use case would be setting up an {{site.data.keyword.IBM_notm}} watsonx platform ready to use in one or more Enterprise accounts granting administrator access
+to an AI Researcher for example.
+It enables an administrator to install all the services making up the {{site.data.keyword.IBM_notm}} watsonx platform, and automatically
+setup an {{site.data.keyword.IBM_notm}} watsonx project, so that the AI Researcher can log into the [platform](http://dataplatform.cloud.ibm.com/wx/home?context=wx) and start working.
+
+In more advanced use cases, the deployable architecture can be used as part of a larger solution, where it is included in a stack
+of other components. For example a solution that first setup the {{site.data.keyword.IBM_notm}} watsonx platform, and then
```
Comment (hiltol) [inline reference-architectures/watsonx-saas-da.md:92]: where it is included in a stack with other deployable architectures. For example, this deployable architecture can be used to first setup the {{site.data.keyword.IBM_notm}} watsonx platform as a foundation, and then another deployable architecture can install an "AI application" that uses the underlying services provisioned by the previous one. To facilitate...

File: reference-architectures/watsonx-saas-da.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+installs a *chatbot application* that uses the underlying services.
+To facilitate those business challenges, the Watsonx AI SaaS deployable architecture provides output parameters that can be used programmatically
+for wiring it to the other components of the stack, and it provides the capability to install additional Watson services.
```
Comment (hiltol) [inline reference-architectures/watsonx-saas-da.md:95]: Watson services -> watsonx services

File: reference-architectures/watsonx-saas-da.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+## Architecture diagram
+{: #architecture-diagram}
+
+![Architecture diagram for the Watsonx AI SaaS Automation with Assistant and Governance deployable architecture](watsonx-saas-da.svg "Architecture diagram for the Watsonx AI SaaS Automation with Assistant and Governance deployable architecture")
```
Comment (hiltol) [inline reference-architectures/watsonx-saas-da.md:100]: There's 2 different names for the DA being used in the doc. Only one of them should be used:
"Watsonx AI Saas deployable architecture" or "Watsonx AI Saas Automation with Assistant and Governance deployable architecture"

File: reference-architectures/watsonx-saas-da.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+{: caption="Figure 1. Watsonx AI SaaS Automation with Assistant and Governance deployable architecture" caption-side="bottom"}{: external download="watsonx-saas-da.svg"}
+
+The Watsonx AI SaaS deployable architecture creates the services shown in the WatsonX services box and an instance of {{site.data.keyword.cos_full_notm}} in a target {{site.data.keyword.Bluemix_notm}} account, resource group, and region. Then, it automatically configures a starter project that grants access to an existing {{site.data.keyword.Bluemix_notm}} user, for example, an AI researcher. As a result, that user can
+log into the {{site.data.keyword.IBM_notm}} watsonx starter project, and start working.
```
Comment (hiltol) [inline reference-architectures/watsonx-saas-da.md:104]: start working -> begin working

File: reference-architectures/watsonx-saas-da.md [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+
+You can add or deploy one or more of the services from the optional services section at a later date during the lifetime of the deployable architecture.
```
Comment (hiltol) [inline reference-architectures/watsonx-saas-da.md:106]: Additional watsonx services from the optional section can be installed at any time after initial deployment of the deployable architecture.

**Review by hiltol — APPROVED** [review]

File: reference-architectures/watsonx-saas-da.md [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+the toc attributes on the H1, repeating the values from the YAML header.
+ -->
+
+The Watsonx AI SaaS deployable architecture is designed to automate the deployment and
+configuration of the IBM watsonx platform in an IBM Cloud account.
+The IBM watsonx platform is made of several services working together to offer AI capabilities to end users,
+who can explore them using [IBM watsonx projects](https://dataplatform.cloud.ibm.com/docs/content/wsj/manage-data/manage-projects.html?context=wx&audience=wdp).
+On that purpose, the automation also configures a *starter project* for an existing IBM Cloud user.
+
+A typical use case would be setting up an IBM watsonx platform ready to use in one or more Enterprise accounts granting administrator access
```
[the remark this answers is not in the document]
Comment (mkrudele) [reply reference-architectures/watsonx-saas-da.md]: These are the two main use cases of the DA. It is quite straightforward, just provision the platform for a team, or provision it in a more sophisticated solution.

File: reference-architectures/watsonx-saas-da.md [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+to install additional Watson services.
+
+## Architecture diagram
+{: #architecture-diagram}
+
+![Architecture diagram for the Watsonx AI SaaS Automation with Assistant and Governance deployable architecture](watsonx-saas-da.svg "Architecture diagram for the Watsonx AI SaaS Automation with Assistant and Governance deployable architecture")
+{: caption="Figure 1. Watsonx AI SaaS Automation with Assistant and Governance deployable architecture" caption-side="bottom"}{: external download="watsonx-saas-da.svg"}
+
+
+The Watsonx AI SaaS deployable architecture lays down the services shown in the `WatsonX services` box and an instance of Cloud Object Storage, in a target IBM Cloud account, Resource Group, and Region. Then, it automatically configures a *starter project* granting
```
[the remark this answers is not in the document]
Comment (mkrudele) [reply reference-architectures/watsonx-saas-da.md]: I changed to `create`.

File: reference-architectures/watsonx-saas-da.md (hunk 8 above)
Comment (mkrudele) [reply reference-architectures/watsonx-saas-da.md:95]: They are not Watsonx services, they are classic Watson services, so I'll leave it as it is.

**Review by hlucey — APPROVED** [review]

**Review by andreainnocenti — APPROVED** [review]

**Pull request merged**